3.8 Docs (#11102)
apollographql/apollo-client: docs/source/api/react/hooks-experimental.mdx

description: Apollo Client experimental react hooks API reference
---

> There are currently no experimental hooks in the latest minor version of Apollo Client, `3.8`. Please see the [Hooks page](./hooks) for a list of available stable React hooks.

Reviewer: ```suggestion
The latest minor version of Apollo Client (`3.8`) has no experimental hooks. Please see the [Hooks page](./hooks) for a list of available stable React hooks.
```

Author: Resolved in https://github.com/apollographql/apollo-client/pull/11136/commits/945b71a1f76e632a5267e5c3b89e9a3764967e4d
